sql-update/murica.sql

```diff
@@ -0,0 +1,4 @@
+update "countries"
+set "name" = 'Murica'
+where "countryId" = 103
```

Comment:
For this, assume the only information you know ahead of time is the old country name:

---

![image](https://user-images.githubusercontent.com/16143771/160739025-a3ebe204-0534-4130-9696-871663432f5f.png)